tests/integration/output/local_log/prepare_MotEvo_parameters.benchmark.log

```diff
@@ -0,0 +1,2 @@
+s	h:m:s	max_rss	max_vms	max_uss	max_pss	io_in	io_out	mean_load
```

Comment:
This is a log file with execution time - it should not be under version control. Miliseconds of execution might change between pipeline runs but that is OK.

Reply:
The same applies to two streams below: empty `stderr` and `stdout`